src/Files.App/Actions/Navigation/OpenDirectoryInNewTabAction.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Files.App.Commands;
+using Files.App.Contexts;
+
+namespace Files.App.Actions
+{
+	internal class OpenDirectoryInNewTabAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		private readonly MainPageViewModel _mainPageViewModel;
+
+		public string Label
+			=> "OpenDirectoryInNewTab".GetLocalizedResource();
+
+		public string Description
+			=> "OpenDirectoryInNewTabDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(opacityStyle: "ColorIconOpenInNewTab");
+
+		public bool IsExecutable =>
```

Comment:
Is there ever a case where this should be disabled? I think we can default to true.

Reply:
Added HasSelection and removed others.